lib/rspec/matchers/built_in/change.rb

```diff
@@ -359,16 +398,7 @@ def actual_delta
       private
 
         def evaluate_value_proc
-          value_proc = @value_proc || lambda { @receiver.__send__(@message) }
-
-          case val = value_proc.call
```

Comment:
glad we could get rid of this

Reply:
Me too.  The dup'ing always felt dirty :(.